aviatrix/resource_aviatrix_gateway.go

```diff
@@ -91,6 +91,19 @@ func resourceAviatrixGateway() *schema.Resource {
 				Computed:    true,
 				Description: "A name for the ELB that is created.",
 			},
+			"vpn_protocol": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Default:     "tcp",
+				Description: "Elb protocol for VPN gateway with elb enabled. Only supports AWS provider. Valid values: 'tcp', 'udp'. Default value: 'tcp'.",
```

Comment:
Set default to "UDP" Since ELB is disabled by default.  Also use "UDP"/"TCP" in the supported values instead of lower case